app/views/users/service_providers/show.html.slim

```diff
@@ -8,29 +8,41 @@ h1 = t('headings.service_providers.show', name: service_provider.name)
 
 table.horizontal-headers
   tr
-    th Issuer
-    td.issuer = service_provider.issuer
-  tr
-    th Name
-    td.name = service_provider.name
+    th Friendly Name
+    td.friendly_name = service_provider.friendly_name
   tr
     th Description
     td.description = service_provider.description
   tr
-    th Metadata URL
-    td.metadata_url = service_provider.metadata_url
+    th Issuer
+    td.issuer = service_provider.issuer
+  tr
```

Comment:
Might be worth re-ordering Agency and Friendly Name. The Agency is the default value on the IdP, which Friendly Name overrides just in UI. Putting Agency first would emphasize that, IMO.

Also, IdP treats Friendly Name as optional but not Agency. Should we put a not-null constraint on Agency?